web/templates/web/resources.html

```diff
@@ -11,7 +11,7 @@
 {% endblock title %}
 {% block content %}
     {% page_title "Resources" %}
-    <h3 class="subheading">Find a developmental lab near you</h3>
+    {% trans '<h3 class="subheading">Find a developmental lab near you</h3>' %}
     <div class="m-4">
```

Comment:
Sorry, I was taking another look at this PR and I noticed that we have HTML in the translations block.  It might be easier for translation if there wasn't HTML in the translation files.

Reply:
Sure, done that!

Reply:
Thank you.